src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +41,34 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{selectedTodo?.id}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              type="button"
+              className="delete"
+              data-cy="modal-close"
+              onClick={() => close()}
```

Comment:
The naming convention for event handler functions suggests using the 'handle' prefix. Consider renaming the 'close' function to 'handleClose' to follow this convention.